### ISSUE TITLE
List un-mastered dungeons + details

### ISSUE THREAD

**Issue by karryt** [cppisking/ffrk-inspector#82]
Not sure if the server sends this down to the client, but I'm guessing it does. If so hopefully it's all at once too and not only when entering a specific game realm. If we could get a list of all unmastered dungeons + the associated stamina completions (and even rewards if it is sent) that would be very helpful. If the data isn't sent all at once though it's not quite as helpful as we'd still need to enter each realm separately, so may not make sense to implement.

**Comment by cppisking** [comment]
If you're using the newest version of inspector, click on the debugging tab
and look for a message that gets sent right when you start FFRK called
/dff/.  That's where all the app initialization data is sent.  You can use
the tree browser on the right hand side to easily navigate through it.  I
know it sends info about available worlds right at startup.  I don't think
it sends information about individual dungeons though, but you could check.

**Comment by karryt** [comment]
I will take a look, thanks! Will finish at least a first pass at moving the install instructions to GH markdown first though :) FIrst time using all of this stuff so I didn't just jump in. Was trying to figure out how to put in the images and then realized you just linked them to imgur which I'll also do. Had to do a little trial and error there too though, haha.

**Comment by cppisking** [comment]
I think you can actually upload them to a post too, but I'm not 100% sure.
Another option is adding them to the repo and then linking to them from
within the document that way.

**Comment by karryt** [comment]
Yeah, putting them in the repo was the first thing I was trying, but I figured if we can just link it off imgur (which I'm guessing is fairly stable?) then I won't need to clutter up the repo for some pictures.

**Comment by cppisking** [comment]
It's stable, but the images wont' last forever on imgur, I think they get
deleted after some time, and also it makes everything versioned more nicely
I guess if the images are in the repo.  It's not a big deal either way
though, just seems kinda nice in a purist sense for everything to be
self-contained.

**Comment by karryt** [comment]
Okay, I lied and just took a quick look at the data that is sent for the game and the stage completion info is sent only when loading each of the 1-13 realms :(  So incomplete stages would only show upon entry unless there was tracking for the users put in place, but I don't think that's appropriate at this point.
